.github/services/s3/ceph_rados_s3/action.yml

```diff
@@ -0,0 +1,46 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: ceph_rados_s3
+description: 'Behavior test for CEPH OBJECT GATEWAY S3.'
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Ceph Rados Server
+      shell: bash
+      working-directory: fixtures/s3
+      run: |
+        docker network create --driver bridge ceph-cluster-net
+        docker compose -f docker-compose-ceph-rados.yml up -d
+    - name: Set up AWS CLI
+      shell: bash
+      run: |
+        echo "AWS_ACCESS_KEY_ID=ceph_rados_s3_admin" >> $GITHUB_ENV
+        echo "AWS_SECRET_ACCESS_KEY=ceph_rados_s3_admin" >> $GITHUB_ENV
+        echo "AWS_DEFAULT_REGION=us-east-1" >> $GITHUB_ENV
+    - name: Setup
+      uses: 1password/load-secrets-action@v1
+      with:
+        export-env: true
+      env:
+        OPENDAL_S3_ROOT: op://services/s3/root
```

Comment:
ceph rados doesn't need those credential from 1password. They should be values that setup in `Setup Ceph Rados Server`.